Homework/Duy_Truong/304_range_sum_query_2D.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @param {number[][]} matrix
+ */
+var NumMatrix = function(matrix) {
+    this.numMatrix = [];
+    for(let i=0;i<matrix.length;i++) {
+        this.numMatrix.push(matrix[i])
+    }
+};
+
+/** 
+ * @param {number} row1 
+ * @param {number} col1 
+ * @param {number} row2 
+ * @param {number} col2
+ * @return {number}
+ */
+NumMatrix.prototype.sumRegion = function(row1, col1, row2, col2) {
```

Comment:
can improve this to O(1) for each query?

Reply:
I've resolve it